python/py-luma.core/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-luma.core
+github.setup        rm-hull luma.core 2.4.1
+github.tarball_from archive
+revision            0
+categories          python
+maintainers         nomaintainer
+platforms           any
```

Comment:
this should probably be `darwin any` as on macOS it uses a Framework build

Reply:
My mental model for this for Python modules has generally been “if it doesn’t install a `.so`, it’s `platform any`. I don’t see a `.so` for any of the modules I’m adding here. Do you see something different, or have a different heuristic in mind?

Reply:
Oh, it occurs to me that this isn’t about `.so` modules, but about the `Python.framework` path that things are installed at. If that’s the case, then I’ll update to `{darwin any}`.

Reply:
> Oh, it occurs to me that this isn’t about `.so` modules, but about the `Python.framework` path that things are installed at. If that’s the case, then I’ll update to `{darwin any}`.

Correct. The point here is indeed not about `.so` files, that is indeed covered by the `noarch` statement. The `platforms` here is for the Python Framework installation that only is the case on darwin. See [this thread on the mailinglist](https://lists.macports.org/pipermail/macports-dev/2022-October/044765.html) where Josh explains this when it was introduced in MacPorts 2.8.0,